### PR TITLE
chore(deps)!: update stylelint to v16

### PR DESCRIPTION
**Pull request by kalievsky**
remove deprecated in v15 rules. See https://stylelint.io/migration-guide/to-15/#deprecated-stylistic-rules

Upgrade major version for both packages

### CODE REVIEW

File: packages/stylelint-config-vue/stylelint.config.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ module.exports = {
   ],
   rules: {
     'scss/dollar-variable-empty-line-before': null,
+    'scss/load-partial-extension': null,
```
Comment (kalievsky) [inline packages/stylelint-config-vue/stylelint.config.js:9]: Not sure. Sometimes we import styles from node_modules by package name only

Comment (kalievsky) [reply packages/stylelint-config-vue/stylelint.config.js:9]: But it could be imported with ignore comment. And leave this rule globally

**Review by deathmaz — CHANGES_REQUESTED** [review]
@kalievsky CI is failing, locally I also getting an error when running `yarn install`:
```sh
➤ YN0000: ┌ Resolution step
➤ YN0001: │ Error: @convidera-team/stylelint-config-convidera@npm:^3.0.0: No candidates found
    at ge (/home/maz/projects/frontend-kit/.yarn/releases/yarn-3.5.0.cjs:439:7784)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Promise.allSettled (index 0)
    at async to (/home/maz/projects/frontend-kit/.yarn/releases/yarn-3.5.0.cjs:390:10398)
➤ YN0000: └ Completed in 1s 539ms
➤ YN0000: Failed with errors in 1s 551ms

```

**Comment by kalievsky** [comment]
Do I maybe need to create dedicated PRs for upgrading of each separate package?

**Comment by deathmaz** [comment]
> Yep. I need you help for this too. There are upgrades of 2 packages. And second one requires new version of first one
> 
> ```
> "@convidera-team/stylelint-config-convidera": "^3.0.0"
> ```
> 
> But it's not published yet. I did not completely get the publishing rules

There's no need to update packages versions by hand, the versions will be updated by `lerna`, it also will update those versions in all dependent packages

**Review by deathmaz — APPROVED** [review]

**Pull request merged**